.github/workflows/autodocs-images.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build Images Reference Docs
+on:
+  workflow_dispatch:
+env:
+  WORKDIR: images
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out Destination Repo"
+        uses: actions/checkout@v2
+        with:
+          path: edu
+
+      - name: "Check out Images Monorepo"
+        uses: actions/checkout@v2
+        with:
+          repository: chainguard-images/images
+          path: images
+
+      - name: "Set up workdir"
+        run: mkdir -p "${{ github.workspace }}/${{ env.WORKDIR }}" && chmod 777 "${{ github.workspace }}/${{ env.WORKDIR }}"
+
+      - name: "Update the reference docs for Chainguard Images"
+        uses: chainguard-dev/deved-autodocs@1.0.2
+        with:
+          command: build images
+        env:
+          YAMLDOCS_SOURCE: "${{ github.workspace }}/images/images"
+          YAMLDOCS_OUTPUT: "${{ github.workspace }}/${{ env.WORKDIR }}"
+
+      - name: "Copy updates to main repo"
+        run: |
+          echo "Copying files..." && \
+          cp -R "${{ github.workspace }}/${{ env.WORKDIR }}" "${{ github.workspace }}/edu/content/chainguard/chainguard-images" && \
+          echo "Finished copy"
+
+      - name: Create a PR
+        uses: peter-evans/create-pull-request@v4
```

Comment:
```suggestion
        uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04 # v4.2.3
```

Reply:
This action looks like magic! Does it need a github token anywhere to create the PR or does it just figure it out for public repos?

Reply:
You can set the token, but by default it uses the credentials from the GitHub bot, that's why it shows up as PR author even thou the commit comes from me. One issue we'll have is with the Enforce app, because it won't be gitsigned, but I think there is an action for gitsign that we could use.